src/main.jsx

```diff
@@ -3,10 +3,13 @@ import ReactDOM from 'react-dom/client';
 import { App } from './pages';
 import './index.css';
 import { ChakraProvider } from '@chakra-ui/react';
+import { ColorModeScript } from './theme/colorModeButton';
```

Comment:
export component from theme folder (using index.js) and name of the file should be the same as the component that's being exported from it

Reply:
index.js has been created and the imports have been shoretened